README.md

```diff
@@ -71,14 +71,14 @@ The `samlconf` script may take the following parameters:
            are optional and if they are not provided, the default values will use DDF's parameters.
     
     OPTIONS
-           -i path
-                The path to the directory containing the implementation's plugin and metadata.
-                The default value is `/implementations/ddf`.
-                      
            -d
                 Boolean for whether or not to enable debug mode which enables more logging.
                 The default value is false.
 
+           -i path
+                The path to the directory containing the implementation's plugin and metadata.
+                The default value is /implementations/ddf.
```

Comment:
❓ Did you purposely get rid of the backticks?

Reply:
```
Yes, because a `backtick` doesn't do anything inside the ``` blocks
```